hmda/src/main/scala/hmda/validation/rules/lar/validity/common/V631_1.scala

```diff
@@ -26,6 +26,15 @@ object V631_1 extends EditCheck[LoanApplicationRegister] {
 
   override def apply(lar: LoanApplicationRegister): ValidationResult =
     when(lar.coApplicant.ethnicity.otherHispanicOrLatino is empty) {
-      lar.coApplicant.ethnicity.ethnicity1 is containedIn(validEthnicities)
```

Comment:
For some strange reason, when the validation function was wrapped under the dsl validation check, it would return ValidationSuccess when the co-applicant's ethnicity1 was any invalid code. So I changed it to explicitly using List.contains() and it worked then. All the tests passed but @PatrickGoRaft might want to verify that this doesn't break anything further.